sheetScript.gs

```diff
@@ -200,37 +200,40 @@ function updatePlaylists(sheet) {
   }
 }
 
-function addVideosToPlaylist(playlistId, videoIds) {
-  var errorCount = 0;
+function addVideosToPlaylist(playlistId, videoIds, idx = 0, successCount = 0, errorCount = 0) {
   var totalVids = videoIds.length;
-  if (videoIds.length < maxVideos) {
-    for (var i = 0; i < totalVids; i++) {
-      try {
-        YouTube.PlaylistItems.insert({
-          snippet: {
-            playlistId: playlistId,
-            resourceId: {
-              videoId: videoIds[i],
-              kind: 'youtube#video'
-            }
+  var success = 1;
```

Comment:
For consistency purposes, can we increment `successCount` at the end of `try` block just like we do with `errorCount`?